web/crux-ui/src/models/container.ts

```diff
@@ -620,3 +626,40 @@ export const mergeJsonConfigToContainerConfig = (
 
   return config
 }
+
+const portToString = (port: ContainerPort): string => `${port.internal}->${port.external}`
+
+export const containerPortsToString = (ports: ContainerPort[], truncateAfter: number = 2): string => {
```

Comment:
Do you mean `portRangetoString()`?

Reply:
this is part of the deployment delete stuff, however i meant ports, cause the docker api sends only list of bound ports instead of ranges so

Reply:
Ehh, okey.